tests/disaster_recovery_2.py

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env python3
+import os
+import shutil
+import signal
+import time
+from TestHarness import Cluster, TestHelper, Utils, WalletMgr
+from TestHarness.Node import BlockType
+
+###############################################################
+# disaster_recovery - Scenario 2
+#
```

Comment:
I like the detailed description here. It will be even better for each of scenarios to have one sentence describing what the purpose of the test.

I think our new tests should follow this test's pattern.